packages/runtime/src/useCases/transport/publicRelationshipTemplateReferences/PublicRelationshipTemplateReferenceMapper.ts

```diff
@@ -0,0 +1,8 @@
+import { PublicRelationshipTemplateReference } from "@nmshd/transport";
+import { PublicRelationshipTemplateReferenceDTO } from "../../../types/transport/PublicRelationshipTemplateReferenceDTO";
+
+export class PublicRelationshipTemplateReferenceMapper {
+    public static toPublicRelationshipTemplateReferencDTO(tagList: PublicRelationshipTemplateReference): PublicRelationshipTemplateReferenceDTO {
```

Comment:
Typo, the last "e" of reference is missing: Write "toPublicRelationshipTemplateReferenceDTO" instead of "toPublicRelationshipTemplateReferencDTO".